internal/sysutil/sysutil.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build !windows, !freebsd
```

Comment:
As a heads up, this wouldn't be right. Both commas and spaces are significant, so you shouldn't use both. See https://godoc.org/go/build#hdr-Build_Constraints:

> A build constraint is evaluated as the OR of space-separated options; each option evaluates as the AND of its comma-separated terms